decidim-admin/app/packs/stylesheets/decidim/admin/_redesigned_settings-menu.scss

```
@@ -1,24 +1,41 @@
 .settings-menu {
```

Related to my other comment, we should change the references from `settings-menu` to `sidebar-menu`.
```suggestion
.sidebar-menu {
```

Also, we should rename this file (to `decidim-admin/app/packs/stylesheets/decidim/admin/_redesigned_sidebar-menu.scss`)

---

Thanks for Suggestion, it has been fixed as part of [bcba67a](https://github.com/decidim/decidim/pull/11654/commits/bcba67a640612446417a510dcacd08c24968a5f0).